.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.6/apache-maven-3.8.6-bin.zip
+wrapperUrl=https://repo.maven.apache.org/maven2/org/apache/maven/wrapper/maven-wrapper/3.1.0/maven-wrapper-3.1.0.jar
```

Comment:
added maven wrapper, as we could not rely that maven is installed in the very same version as on your system.

Reply:
Shouldn't both URI's have the same version?

Reply:
Interessting, I just took it from what the spring initiallizer created. I would believe that this is ok. Both are different products. So they must not necessarily be in the same version, I would think.